.github/workflows/opensearch-builder.yaml

```
@@ -0,0 +1,69 @@
+        uses: actions/checkout@v4
+
+      - name: Build image for PR
+        if: ${{ github.event_name == 'pull_request' }}
+        env:
+          CRC_BUILDER: ghcr.io/crc-org/ci-opensearch
+          CRC_BUILDER_V: pr-${{ github.event.number }}
+        run: |
+          make opensearch-oci-build
+          make opensearch-oci-save
+          echo "image=${CRC_BUILDER}:${CRC_BUILDER_V}" >> "$GITHUB_ENV"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)